cg/meta/upload/scout/scout_config_builder.py

```diff
@@ -27,24 +40,101 @@ def __init__(self, hk_version_obj: Version, analysis_obj: Analysis, lims_api: Li
         self.load_config: ScoutLoadConfig = ScoutLoadConfig(
             delivery_report=self.get_file_from_hk({HK_DELIVERY_REPORT_TAG})
         )
```

Comment:
Can `self.load_config` be removed now since we create the object in `build_load_config`?

Reply:
Removed!